src/pages/Transaction/TransactionComp/TransactionLite/TransactionLite.tsx

```diff
@@ -15,18 +15,13 @@ export const TransactionCompLite: FC<{ isCellbase: boolean }> = ({ isCellbase })
   const { hash: txHash } = useParams<{ hash: string }>()
   const [t] = useTranslation()
 
-  const query = useQuery(
-    ['ckb_transaction_details', txHash],
-    async () => {
-      const ckbTransactionDetails = await explorerService.api.fetchTransactionLiteDetailsByHash(txHash)
-      return ckbTransactionDetails.data
-    },
-    {
-      initialData: defaultTransactionLiteDetails,
-    },
+  const { data: _transactionLiteDetails } = useQuery(['ckb_transaction_details', txHash], () =>
+    explorerService.api.fetchTransactionLiteDetailsByHash(txHash).then(res => res.data),
   )
 
-  const txList = query.data.map(tx => ({
+  const transactionLiteDetails = _transactionLiteDetails ?? defaultTransactionLiteDetails
```

Comment:
`defaultTransactionLiteDetails` can be set as the default value of `transactionLiteDetails` to avoid this nullish coalescing